pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/codestyle/xml/UnnecessaryLocalBeforeBranch.xml

```diff
@@ -356,4 +356,221 @@ public class Example {
 }
 ]]></code>
     </test-code>
-</test-data>
+    <test-code>
+        <description>Simple unnecessary local before throw</description>
```

Comment:
```suggestion
        <description>#5770 Simple unnecessary local before throw</description>
```

We try to reference the github issue from which the test case originated

Reply:
thx.